src/Dynatrace.MetricUtils/DynatraceMetricApiConstants.cs

```diff
@@ -37,6 +39,7 @@ public static class DynatraceMetricApiConstants
 		public static int PayloadLinesLimit => LinesLimit;
 
 		/// <summary>The maximum number of dimensions that can be attached to one metric key.</summary>
+		[Obsolete("Dimensions are no longer dropped.")]
```

Comment:
```suggestion
		[Obsolete("All dimensions passed to the serializer are added")]
```

Nit: Maybe something like this instead?